setup.cfg

```diff
@@ -1,2 +1,2 @@
 [metadata]
-license_file = LICENSE
+license_files = LICENSE
```

Comment:
this is the only change that's giving me pause. I've fallen of the train of the situations under which this doesn't work as intended (old setup tools?) but it would be a straightforward change to back it out if it causes troubles.